src/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const Storage = {
+    get(key: string) {
+        const data = localStorage.getItem(key)
+
+        return JSON.parse(data)
+    },
+    set(key: string, data: Array<Object>) {
```

Comment:
I don't think so, you can save everything, so basically it should be `any`. Any can be used, but only if it makes sense, not if you don't know the type or you are lazy 😄